JavaScript/React.md

```diff
@@ -1,14 +1,17 @@
 # React Guidelines and Best Practices
```

Comment:
[There are no best practices](https://www.satisfice.com/blog/archives/5164) 😬

Reply:
Didn't read the whole article yet, but I'm not sure they're talking about practices or rules - because one is a recommendation and the other is a must. We make this clear with the first sentence after the title :D

Reply:
I was referring to the fact that there are no _best practices_ because there are no universal best practices that all should abide by.
Just the recommendations in a given situation that yield the most benefit 🙂 

I'd rename the chapter to something like 

`React Guidelines and Recommended Infinum Practices`